thentos-core/src-devel/Paths_thentos_core.hs

```diff
@@ -0,0 +1,6 @@
+-- | data files and development mode.  for details, see:
+-- http://neilmitchell.blogspot.de/2008/02/adding-data-files-using-cabal.html
+module Paths_thentos_core where
+
+getDataFileName :: FilePath -> IO FilePath
+getDataFileName = return
```

Comment:
What's the connection to the rest of the PR? And are we sure this is doing the right thing in non-development mode? I don't see how it'd be able to behave differently.